src/Giraffe/HttpHandlers.fs

```diff
@@ -351,31 +351,34 @@ let renderHtml (document : XmlNode) : HttpHandler =
 /// Example:
 /// let negotiationRules = dict [ "application/json", json; "application/xml" , xml ]
 /// `json` and `xml` are both the respective default HttpHandler functions in this example.
-let negotiateWith (negotiationRules    : IDictionary<string, obj -> HttpHandler>)
+let negotiateWith (negotiationRules    : #IDictionary<string, obj -> HttpHandler>)
```

Comment:
What does the `#` do here? (noob question)

Reply:
it allows implicit acceptance of interface without need to cast the type to the interface (basically)

Reply:
it means "flexible types" and makes F# accept "more" dictionaries than before. I generally see it as good habit, while it probably does not change anything here. Type is already fixed to string * obj -> HTTPHandler. I revert that change

Reply:
Thanks for the explanation!